okhttp/src/main/java/okhttp3/internal/http/HttpDate.kt

```diff
@@ -13,40 +13,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package okhttp3.internal.http;
+package okhttp3.internal.http
 
-import java.text.DateFormat;
-import java.text.ParsePosition;
-import java.text.SimpleDateFormat;
-import java.util.Date;
-import java.util.Locale;
-
-import static okhttp3.internal.Util.UTC;
+import okhttp3.internal.Util.UTC
+import java.text.DateFormat
+import java.text.ParsePosition
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.Date
 
 /**
  * Best-effort parser for HTTP dates.
  */
-public final class HttpDate {
-  /** The last four-digit year: "Fri, 31 Dec 9999 23:59:59 GMT". */
-  public static final long MAX_DATE = 253402300799999L;
+object HttpDate {
+  /** The last four-digit year: "Fri, 31 Dec 9999 23:59:59 GMT".  */
+  const val MAX_DATE = 253402300799999L
 
   /**
    * Most websites serve cookies in the blessed format. Eagerly create the parser to ensure such
    * cookies are on the fast path.
    */
-  private static final ThreadLocal<DateFormat> STANDARD_DATE_FORMAT =
-      new ThreadLocal<DateFormat>() {
-        @Override protected DateFormat initialValue() {
-          // Date format specified by RFC 7231 section 7.1.1.1.
-          DateFormat rfc1123 = new SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss 'GMT'", Locale.US);
-          rfc1123.setLenient(false);
-          rfc1123.setTimeZone(UTC);
-          return rfc1123;
-        }
-      };
+  private val STANDARD_DATE_FORMAT = object : ThreadLocal<DateFormat>() {
+    override fun initialValue(): DateFormat {
+      // Date format specified by RFC 7231 section 7.1.1.1.
+      val rfc1123 = SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss 'GMT'", Locale.US)
```

Comment:
Just throwing this out here. But I'm not sure what's more readable.
```suggestion
      val rfc1123 = SimpleDateFormat("EEE, dd MMM yyyy HH:mm:ss 'GMT'", Locale.US).apply {
          isLenient = false
          timeZone = UTC
      }
```

Reply:
Agree, it's a good use-case for `apply`

Reply:
6b9b76143563db1b3de49dd87348682afd891542